conn-fix/README.md

```diff
@@ -0,0 +1,359 @@
+# FIX Connect
+
+This "Connect" component extension supports connection to the target system via [FIX protocol](https://www.fixtrading.org/what-is-fix/).
+
+## Available service types
+
+This "Connect" component provides users with two types of the services:
+
+### FIX Client
+
+The "Connect" component acts like a client that connects to the system.
+The configuration type that needs to be used is - **FIX_Client**.
+
+#### Configurable parameters
+
+The FIX Client has the following parameters to configure:
+```yaml
+# It is a required parameter. It's the FIX version that this session should use.
+# Possible values: "FIX.4.0", "FIX.4.1", "FIX.4.2", "FIX.4.3", "FIX.4.4", "FIXT.1.1"
+beginString: FIXT.1.1
+
+# It is a required parameter. Your ID that is associated with this FIX session.
+senderCompID: sender
+
+# It is a required parameter. Counter Parties ID that is associated with this FIX session.
+targetCompID: target
+
+# It is a required parameter. Directory to store sequence number and message files inside the workspace directory.
+fileStorePath: store/fix/sessions
+
+# It is a required parameter. Time of day that this FIX session becomes activated.
+# time in the format of HH:MM:SS, time is represented in UTC.
+# If the session creation time is not between the StartTime and the EndTime, the session state will be reset (incoming / outgoing sequences and message cache will be drop).
+# The bounds move daily. So it means that, in the next day after the start at 'EndTime' the bounds will be moved
+# Format of the value is 99:99:99
+startTime: 00:00:00
+
+# It is a required parameter. Time of day that this FIX session becomes deactivated.
+# time in the format of HH:MM:SS, time is represented in UTC.
+# If the session creation time is not between the StartTime and the EndTime, the session state will be reset (incoming / outgoing sequences and message cache will be drop).
+# The bounds move daily. So it means that, in the next day after the start at 'EndTime' the bounds will be moved
+# Format of the value is 99:99:99
+endTime: 00:00:00
+
+# It is a required parameter.
+# Specifies the default application version ID for the session.
+# This can either be the ApplVerID enum (see the ApplVerID field) or the BeginString for the default version.
+# The possible values are: "2", "3", "4", "5", "6", "7", "8", "9"
+defaultApplVerID: 9
+
+# Determines if the sequence numbers should be reset to 1, after a normal logout termination
+resetOnLogout: false
+
+# Determines if the sequence numbers should be reset to 1, after an abnormal termination
+resetOnDisconnect: false
+
+# Save Heartbeats to logs or not.
```

Comment:
**If this parameter is set to true, it can** save Heartbeats to logs.

Reply:
I'm not sure about can, maybe use will in this case?